src/client/game/game.js

```diff
@@ -3,33 +3,79 @@ import { connect } from 'react-redux';
 
 import AnswerInput from './answer-input';
 
-import { setLeader } from './game-actions';
+import { setLeader, getConsonantDispatch, getVowelDispatch, 
+        disableConsonantDispatch, disableVowelDispatch } from './game-actions';
 
-import { setLeaderDispatch } from '../services/cloak-service';
+import { setLeaderDispatch, setConsonantDispatch, messageGetConsonant, 
+        setVowelDispatch, messageGetVowel, setDisableConsonantDispatch,
+        setDisableVowelDispatch } from '../services/cloak-service';
+
+import storageService from '../services/storage-service';
 
 export class Game extends Component {
     componentWillMount() {
         setLeaderDispatch(this.props.setLeader);
+        setConsonantDispatch(this.props.getConsonantDispatch);
+        setVowelDispatch(this.props.getVowelDispatch);
+        setDisableConsonantDispatch(this.props.disableConsonantDispatch);
+        setDisableVowelDispatch(this.props.disableVowelDispatch);
+    }
+
+    isLeader() {
+        let userId = storageService.getUser().name;
+        if(userId === this.props.leader) {
```

Comment:
```
isLeader() {
    let userId = storageService.getUser().name;
    return userId === this.props.leader;
}
```

Also why is it `getUser().name` but the variable is called `userId`. Is it a name or an ID?